backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/nuernberg/steps/Store.kt

```diff
@@ -1,60 +0,0 @@
-package app.ehrenamtskarte.backend.stores.importer.nuernberg.steps
-
-import app.ehrenamtskarte.backend.projects.database.ProjectEntity
-import app.ehrenamtskarte.backend.projects.database.Projects
-import app.ehrenamtskarte.backend.stores.database.AcceptingStoreEntity
-import app.ehrenamtskarte.backend.stores.database.AddressEntity
-import app.ehrenamtskarte.backend.stores.database.Categories
-import app.ehrenamtskarte.backend.stores.database.ContactEntity
-import app.ehrenamtskarte.backend.stores.database.PhysicalStoreEntity
-import app.ehrenamtskarte.backend.stores.importer.ImportConfig
-import app.ehrenamtskarte.backend.stores.importer.PipelineStep
-import app.ehrenamtskarte.backend.stores.importer.common.types.AcceptingStore
-import net.postgis.jdbc.geometry.Point
-import org.jetbrains.exposed.dao.id.EntityID
-import org.jetbrains.exposed.sql.transactions.transaction
-import org.slf4j.Logger
-
-class Store(config: ImportConfig, private val logger: Logger) :
-    PipelineStep<List<AcceptingStore>, Unit>(config) {
-
-    override fun execute(input: List<AcceptingStore>) {
-        transaction {
-            val project = ProjectEntity.find { Projects.project eq config.findProject().id }.single()
-            try {
-                project.deleteAssociatedStores()
-
-                input.forEach { acceptingStore ->
-                    val address = AddressEntity.new {
-                        street = acceptingStore.streetWithHouseNumber
-                        postalCode = acceptingStore.postalCode!!
-                        location = acceptingStore.location
-                        countryCode = "de"
```

Comment:
I previously thought, that `Store` was the same for both Bayern and Nürnberg, but this hard coded `"de"` is different. Check whether we should hard code it in a previous step.

Reply:
There's no problem: The country code is already hardcoded as  "de" in the previous step "MapFromCSV":
https://github.com/digitalfabrik/entitlementcard/blob/5ca75fc24f4e308426540aa0c9f84f6baa5626ca/backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/nuernberg/steps/MapFromCsv.kt#L35